controllers/datafoundation.go

```diff
@@ -329,3 +334,29 @@ func (r *dataFoundationReconciler) reconcileRookCephOperatorConfig() error {
 
 	return nil
 }
+
+func (r *dataFoundationReconciler) reconcileOCSInitialization() error {
+	r.Log.Info("Reconciling OCSInitialization")
+
+	ocsInitList := ocsv1.OCSInitializationList{}
```

Comment:
- will there be more than one CRs of this type?
- if not, will there be different names for these CRs across deployments?
- IOW, if there's a single CR of this CRD and static name across deployment, why can't just get it directly?

Reply:
> will there be more than one CRs of this type?

No, it will always be a single CR of this type. It is created when OCS-Operator is installed.
> if not, will there be different names for these CRs across deployments?

No, name of this CR will always be ocsinit
> IOW, if there's a single CR of this CRD and static name across deployments, why can't just get it directly?

Yes, updated it in the latest commit.